ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c

```diff
@@ -23,10 +24,13 @@
  * @see		https://github.com/nnstreamer/nnstreamer
  * @author  Jihoon Lee <ulla4571@gmail.com>
  *          niklasjang <niklasjang@gmail.com>
- * @bug		No known bugs except for NYI items
+ *          Dongju Chae <dongju.chae@samsung.com>
+ * @bug     No known bugs except for NYI items
  *
  * option1: Decoder mode of image segmentation
  *          Available : tflite-deeplab
+ *          Available : snpe-deeplab
+ *          Available : snpe-depth
```

Comment:
Use the name of the base model, not the converted model. Both base and converted are supposed to have the same input and output requirements.

Reply:
In my understanding, `tflite-deeplab` and `snpe-deeplab` have different input/output requirements because they have different network backbone.
- `tflite-deeplab`: mobilenet-v2, input size: 257x257
- `snpe-deeplab`: Xception_65, input size: 513x513x3

As you mentioned, I will add input/output requirements.